frontend/src/components/placementVisuals/PlacementSamplesTable.tsx

```diff
@@ -121,7 +121,21 @@ const PlacementSamplesTable = ({ container: containerName, showContainerColumn }
         selectedRowKeys,
         onChange,
         onSelect,
-    }), [selectedRowKeys, onChange, onSelect])
+        getCheckboxProps: (sample) => ({
+            disabled: isDestination && sample.parentContainerName === containerName
+        })
+    }), [selectedRowKeys, onChange, onSelect, isDestination, containerName])
+
+    const paginationProps: NonNullable<TableProps<PlacementSample>['pagination']> = useMemo(() => ({
+        showSizeChanger: true,
+        showTotal(total, range) {
+            return <>
+                <>{`${range[0]}-${range[1]} of ${total} items.`}</>
+                <>{' '}</>
+                <>{`${selectedRowKeys.length} selected.`}</>
```

Comment:
It looks like it takes precious space a button of the pagination (button to go to the next page) wrap around to the next line on my screen. Not sure this indicator is worth breaking the harmony of the display ...

Reply:
So shall I undo this?

Reply:
Up to you ... I don't see a great way to replace this to let people know they selected something if they are not on the first page...